spec/system/module_overview_progress_spec.rb

```diff
@@ -97,28 +100,51 @@
       visit '/modules/alpha'
     end
 
-    it 'the submodule indicator is "in progress"' do
-      within '#section-button-1' do
-        expect(page).to have_content 'in progress'
+    it 'the first topic can be clicked' do
+      within '#section-content-2 .module-section--container .module-section--item:nth-child(1)' do
```

Comment:
Do we need this specificity? Guessing it will still pass with just `#section-content-2` but you are being consistent across the specs, and those which check the indicator do need to be specific.